robolectric/src/main/java/org/robolectric/android/controller/FragmentController.java

```diff
@@ -123,6 +123,11 @@ public void run() {
     return this;
   }
 
+  public FragmentController<F> arguments(Bundle bundle) {
```

Comment:
You might want to make this a second constructor argument, then have an overloaded buildFragment(..., Bundle) method, this way its not possible to call the builder methods out of order. We had that issue on withIntent() for Activities.

Reply:
I don't see any `buildFragment()` methods to overload. Do you mean `create()`?

Reply:
Disregard my comment. Found what you meant. Will look into it.

Reply:
@jongerrish  Updated as per your comment. Let me know if this change was what you requested